link.x

```diff
@@ -2,21 +2,32 @@ INCLUDE memory.x
 
 SECTIONS
 {
-  .text ORIGIN(FLASH) :
+  .vector_table ORIGIN(FLASH) :
   {
     /* Vector table */
-    _VECTOR_TABLE = .;
+    _svector_table = .;
```

Comment:
well the only thing actually offset from the `_svector_table` symbol is the initial SP, right? So, pointer to `_fstack`, so pointer to start of stack. I admit this is a pretty bad initialism if none of you got it :) I have nothing, sure, `_svectors` sounds good enough